pkg/util/github/github.go

```diff
@@ -190,7 +190,9 @@ func DownloadOpenShiftReleaseBinary(binaryType OpenShiftBinaryType, osType minis
 	binaryPath := ""
 	switch {
 	case strings.HasSuffix(assetTmpFile, TAR):
-		tarFile := assetTmpFile[:len(assetTmpFile)-3]
+		tokens := strings.Split(assetFilename, fmt.Sprintf("%s-", version))
```

Comment:
I am reluctant to put anything like this in for now. Let's first wait what the response is from the OpenShift side.

If we need to handle this ourselves, I think we need to write some code which is a bit  better to understand than this.